### PR TITLE
Change ERB.new call to use named arguements

### PR DESCRIPTION
**Pull request by brlanier**
- Fixes depreciation warning about passing positional arguments. ERB depreciated them in recent releases and prefer the named arguments

### CODE REVIEW

**Comment by brlanier** [comment]
Addresses #92

**Comment by brlanier** [comment]
Because the fix goes back pretty far, not sure if we want to do a bigger release version when we get there. Right now, I think we are releasing in `0.1.X` and we could switch to `0.2.X` or maybe go with a full `1.0.0` release.

**Comment by brlanier** [comment]
@javierg Can you take a look at this. I think it is pretty straightforward. We would just need to decide if we are going to change the minor version or finally release a major version or what we want to do here.

File: biran.gemspec [hunk 1]
```diff
@@ -13,6 +13,7 @@ Gem::Specification.new do |s|
   s.summary     = "Helper for generating config generate tasks."
   s.description = "Biran is the guy that will help you generate config files for your rail app."
   s.license     = "MIT"
+  s.required_ruby_version = ">= 2.6.0"
```
[the remark this answers is not in the document]
Comment (javierg) [reply biran.gemspec:16]: for us is fine, though ruby 2.6 is not supported by ruby anymore

**Review by javierg — APPROVED** [review]
:shipit:

File: biran.gemspec (hunk 1 above)
[the remark this answers is not in the document]
Comment (brlanier) [reply biran.gemspec:16]: Yes, this was more to handle any older apps that might not be updated that use biran. This should prevent this version from installing in those instances. 
We can always start a new branch after this to only run and test against supported ruby versions. However, this was a change that has long been needed, so it made sense to me to lock this here.

**Pull request merged**